src/main/java/lotto/service/LottoService.java

```diff
@@ -0,0 +1,68 @@
+package lotto.service;
+
+import lotto.domain.Lotto;
+import lotto.domain.Rank;
+import lotto.domain.Result;
+import lotto.domain.WinningStatistics;
+import lotto.utils.NumberGenerator;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoService {
+    private static final int LOTTO_PRICE = 1000;
+    private static final int DEFAULT_WINNING_COUNT = 0;
+
+    private final NumberGenerator numberGenerator;
+
+    public LottoService(NumberGenerator numberGenerator) {
+        this.numberGenerator = numberGenerator;
+    }
+
+    public List<Lotto> buyLottos(int lottoQuantity) {
+        List<Lotto> lottos = new ArrayList<>(lottoQuantity);
+        for (int currentLottoQuantity = 0; currentLottoQuantity < lottoQuantity; currentLottoQuantity++) {
+            lottos.add(createLotto());
+        }
+
+        return lottos;
```

Comment:
(이렇게 바꿔보는 건 어때요?)

```suggestion
        return Stream.generate(this::createLotto)
                    .limit(lottoQuantity)
                    .toList();
```
이런식으로 스트림을 통해서 간결하게 만드는 방법도 있어요 :)